internal/provider/validators_test.go

```diff
@@ -296,8 +296,8 @@ func Test_validateConsulClusterTier(t *testing.T) {
 			expected: diag.Diagnostics{
 				diag.Diagnostic{
 					Severity:      diag.Error,
-					Summary:       "expected dev to be one of [DEVELOPMENT STANDARD PLUS]",
-					Detail:        "expected dev to be one of [DEVELOPMENT STANDARD PLUS] (value is case-insensitive).",
+					Summary:       "expected dev to be one of [DEVELOPMENT STANDARD PLUS PREMIUM]",
```

Comment:
Updating the go SDK (https://github.com/hashicorp/hcp-sdk-go/pull/164) added this field. Had to fix this test